linux_os/guide/system/software/integrity/fips/is_fips_mode_enabled/rule.yml

```diff
@@ -1,7 +1,7 @@
 documentation_complete: true
 
 
-title: Verify '/proc/sys/crypto/fips_enabled' exists  
+title: Verify '/proc/sys/crypto/fips_enabled' exists
```

Comment:
The mere existence of `/proc/sys/crypto/fips_enabled` is not sufficient. All kernels that support enabling FIPS mode will expose this file in user space. It must contain `1` for FIPS mode to be actually enabled.

Reply:
I have removed this rule from RHEL 10.